src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs

```diff
@@ -166,6 +184,23 @@ public static void ValidateFromBase64_NoPadding(string data)
             }
         }
 
+        [Theory, MemberData(nameof(TestData_Oversize))]
+        public static void ValidateFromBase64_OversizeBuffer(string input, int offset, int count, string expected)
+        {
+            using (var transform = new FromBase64Transform())
+            {
+                byte[] inputBytes = Text.Encoding.ASCII.GetBytes(input);
+                byte[] outputBytes = new byte[100];
+
+                // skip the first and last byte
```

Comment:
This comment doesn't really make sense.  Looking at the commits vector, it looks just to be out of date.